nf_core/pipeline-template/.gitattributes

```diff
@@ -1 +1,3 @@
 *.config linguist-language=nextflow
+modules/nf-core/** linguist-generated
+subworkflows/nf-core/** linguist-generated
```

Comment:
No `=nextflow` here?

Reply:
Um, I think we should blame @grst for this but `linguist-generated` was to ignore particular types of files in the `diff` as documented [here](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github)